src/pages/Profile/MyProfile.jsx

```diff
@@ -1,71 +1,35 @@
+/* eslint-disable */
+
 import React from "react";
 import "./MyProfile.css";
-import profile from "./profile.png";
-import useAuth from "../../hooks/useAuth";
-/* eslint-disable jsx-a11y/label-has-associated-control */
-
-const EditButtons = () => <button type="button" className="editButton">Edit</button>;
+import MyProfileImage from "./MyProfileImage";
+import MyProfilePersonalData from "./MyProfilePersonalData";
+import useGetConnectedUser from "../../hooks/useGetConnectedUser";
 
-const MyProfileAboutYou = () => (
-  <div className="personalInformationContainer">
-    <label htmlFor="title" className="titleInsideContainer">
-      About You
-    </label>
-    <div className="field">
-      <label className="label">Name</label>
-      <p>My Name</p>
-    </div>
+const MyProfile = () => {
+  const [loading, data] = useGetConnectedUser();
 
-    <div className="field">
-      <label className="label">Email</label>
-      <p>My Email</p>
-    </div>
+  if (loading) {
+    return <div>Loading...</div>;
+  }
 
-    <div className="field">
-      <label className="label">Contact</label>
-      <p>My Contact</p>
-    </div>
-    <EditButtons />
-  </div>
-);
+  const myProfile = "Mon Profil";
 
-const MyProfileImage = () => {
-  const { logout } = useAuth();
   return (
-    <div className="photoProfileContainer">
-      <figure className="image is-128x128">
-        <img className="is-rounded" src={profile} alt="Profile" />
-      </figure>
-      <label htmlFor="title" className="titleInsideContainer">
-        First name
-      </label>
-      <button type="button" className="editButton" onClick={logout}>
-        Log Out
-      </button>
-    </div>
+    <>
+      {!loading && (
+      <div className="myProfile">
+        <div className="myProfile__title__container">
+          <h1 id="myProfile__title">{myProfile}</h1>
+        </div>
+        <div className="myProfile__main__section">
+          <MyProfileImage/>
+          <MyProfilePersonalData data={data} />
```

Comment:
- **Proposition 1** : au lieu d'avoir se composant "Personal Data", tu pourrais l'enlever et mettre à la place 2 composants que tu as mis dans MyProfilePersonalData => un composant pour la partie is editing et un composant pour la partie !isEditing 

Donc, dans ton return:
**<MyProfileImage />
{!isEditing ? <!PersonalData /> : <!PersonalDataEdition>}**

**-Proposition 2**: dans la partie d'edition tu pourrais directement utiliser les composants utilisées pour l'authentification, le CSS de ses composants est défini dans le form parent dans Login ou Signup => donc tu peux reutiliser ses composants et mettre le CSS dans un form parent (comme j'ai fait pour Login et SignUp)
**<MyProfileImage />
{isEditing ? <PersonalData> : <form>  <EnterName /> <EnterSurname/> <EnterEmail/> </form>}**

- **Option 3** : tu gardes comme tu as fait, c'est pas déjà pas mal